library/src/main/java/com/eggheadgames/siren/Siren.java

```diff
@@ -231,13 +245,21 @@ private int checkVersionDigit(String[] minVersionNumbers, String[] currentVersio
     private boolean checkVersionCode(JSONObject appJson) throws JSONException{
         if (!appJson.isNull(Constants.JSON_MIN_VERSION_CODE)) {
             int minAppVersionCode = appJson.getInt(Constants.JSON_MIN_VERSION_CODE);
+            // If no config found, assume version check is enabled
+            Boolean versionCheckEnabled = appJson.has(Constants.JSON_ENABLE_VERSION_CHECK) ? appJson.getBoolean(Constants.JSON_ENABLE_VERSION_CHECK) : true;
+            if (!versionCheckEnabled) {
+                return false;
+            }
+
+            // If no config found, assume force update = false
+            Boolean forceUpdateEnabled = appJson.has(Constants.JSON_FORCE_ALERT_TYPE) ? appJson.getBoolean(Constants.JSON_FORCE_ALERT_TYPE) : false;
```

Comment:
I think here it's possible to do the same as with `Constants.JSON_ENABLE_VERSION_CHECK` logic. But for `Constants.JSON_FORCE_ALERT_TYPE`

Reply:
Created #30